### ISSUE TITLE
HTML Entities not rendering in UnicodeIDStart, UnicodeIDContinue

### ISSUE THREAD

**Issue by rwaldron** [tc39/ecma262#931]
```
UnicodeIDStart::
    any Unicode code point with the Unicode property &ldquo;ID_Start&rdquo;
UnicodeIDContinue::
    any Unicode code point with the Unicode property &ldquo;ID_Continue&rdquo;
```

![](https://i.gyazo.com/fb35ce69d6e34411e83e946148e4f23c.png)

https://tc39.github.io/ecma262/#prod-UnicodeIDStart

**Comment by mathiasbynens** [comment]
https://github.com/bterlson/ecmarkup/issues/118
https://github.com/bterlson/ecmarkup/pull/117

**Comment by rwaldron** [comment]
@mathiasbynens ah, thanks! ❤️

**Comment by bterlson** [comment]
I'll try to get to this today, I have discussed a possible fix with @rbuckton that should be fairly easy to implement.

**Comment by bterlson** [comment]
Involving yet another @mathiasbynens's library, of course (in this case, "he"). Idea is in cases where we slice the source text to pass to grammarkdown we first decode HTML entities.

**Comment by bterlson** [comment]
Fixed bterlson/ecmarkup#117.
